roles/openshift_node_group/files/sync.yaml

```diff
@@ -141,6 +141,9 @@ spec:
                 continue
               fi
             fi
+            # annotate node with md5sum of the config
+            oc annotate --config=/etc/origin/node/node.kubeconfig "node/${NODE_NAME}" \
+              node.openshift.io/md5sum="$( cat /tmp/.new | cut -d' ' -f1 )" --overwrite
```

Comment:
I’m not sure we will allow this going forward (the node can’t have credentials that allow them to set this.

Reply:
I think as long as we can assume it continues to work until 4.0, independent of kube version, we're fine under the assumption that this problem is no longer in the domain of openshift-ansible.

Reply:
i do not see us removing the ability to annotate before we move to dynamic kubelet config.